js/main.js

```diff
@@ -47,6 +51,8 @@ function getUrlAndOpenMainWindow() {
         let title = 'Error loading configuration';
         electron.dialog.showErrorBox(title, title + ': ' + err);            
     });
+
+    applyContextMenu();
```

Comment:
Do we plan to have different right click menu based on child or main window?
Current implementation applies the same context menu on main window and on any window created after that.